R/dvhydrograph-render.R

```diff
@@ -8,10 +8,11 @@ createDvhydrographPlot <- function(data) {
   options(scipen=8)
   
   dvData <- parseDVData(data)
```

Comment:
FYI, I'm on a similar path as you and maybe going a step further to break out the uber "dvData" and "dvInfo" objects. Rather than calling one uber parse function, going to try to break the parse function up into smaller parse functions and collect those objects separately to pass to the rendering.  I haven't started yet so I don't know how small it's going to be, but going to look for some natural logical divisions. You might want to continue down that path as well.